python/py-commpy/Portfile

```diff
@@ -20,16 +20,17 @@ python.rootname     scikit-commpy
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  dd1642d3fdbf7ef21dbcb2f58a1732dfdc338696 \
-                    sha256  a3903df1a9af50654f354eaf3371d13406514405d75a7c0f8f4cf5957fac90e0 \
-                    size    26768
+checksums           rmd160  c2ecb0c6582bca88b2bf19a64430058cc83f2c54 \
+                    sha256  5cd94641e0916425e8370a2d4ba36669b4fc1516a96ef4aed79d8a5f8ab1da4e \
+                    size    42700
 
-python.versions     27 35 36 37 38
+python.versions     35 36 37 38
```

Comment:
Instead of entirely removing support for Py27 / v0.4.0, please pit the version like we do in NumPy. Easy peasy change. Even if Py27 is officially EOL, there are plenty of projects that still use it. Not sure about this specific Port, but since it's not difficult to keep supporting it for a bit, let's do so. Thx!

Reply:
Oh; just saw that Renee asked for the same ... maybe great minds think alike? LOL

Reply:
done